ansible/letsencrypt_cert.yml

```diff
@@ -0,0 +1,100 @@
+---
+# commcare-cloud <env> ansible-playbook letsencrypt_cert.yml
+- hosts: proxy
+  become: yes
+  tasks:
+    - name: Add certbot apt repo
+      apt_repository: repo='ppa:certbot/certbo' state=present
+      register: add_cert_repo
+
+    - name: Update package list
+      apt: update_cache=yes
+      when: add_cert_repo|changed
+
+    - name: Install Certbot
+      apt:
+        name: "{{item}}"
+        state: present
+      with_items:
+        - software-properties-common
+        - certbot
+      ignore_errors: '{{ ansible_check_mode }}'
+
+    - name: Create Challenge Directory
```

Comment:
can't `certbot` do all of this?

Reply:
yeah, Certbot can do it in the complete automated way. But I do not want it to temper with our nginx configurations.